src/main/java/seedu/ui/Help.java

```diff
@@ -40,15 +42,17 @@ public void printEventHelp() {
             + "To delete an event, use the command\n"
             + "\tevent delete i/Event_index\n"
         );
+        output.append("Enter 'list' for command format list. To exit help function, enter 'back'\n");
         display(output.toString());
     }
 
     public void printCalendarHelp() {
         display("To view events in a calendar, there should be events that exist in the event list.");
-        display("\tTo view calendar, input: s/Semester_Number ay/Year_one-Year_two.\n");
+        display("To view calendar, input: s/Semester_Number ay/Year_one-Year_two.\n");
         display("* Note that semester can only be an integer 1 or 2.");
         display("* Note that the academic year must contain two consecutive years separated by a hyphen.\n "
                 + " Each year should be in a double digit format.");
+        display("Enter 'list' for command format list. To exit help function, enter 'back'\n");
```

Comment:
Why list under Calendar?

Reply:
this is for command format, now help function requires user to input 'back' to exit

Reply:
this is for command format, now help function requires user to input 'back' to exit